manifests/t/the-sz/Compton/1.51/the-sz.Compton.yaml

```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.singleton.1.0.0.schema.json
+
+PackageIdentifier: the-sz.Compton
+PackageVersion: 1.51

```

Comment:
Package has no version in the ARP entry, this will lead to upgrade loop
![image](https://user-images.githubusercontent.com/12611259/125694303-00f59910-77e4-4a80-867f-312755fd9864.png)

Reply:
fixed with new commit